### PR TITLE
Add common installer for both "AM" and "AppMan"

### PR DESCRIPTION
**Pull request by ivan-hc**
Istantanea_2024-08-03_21-29-46

### CODE REVIEW

**Comment by ivan-hc** [comment]
Those who want to test the new installr, can download it from https://github.com/ivan-hc/AM/blob/dev/AM-INSTALLER

File: AM-INSTALLER [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ ##########################################################################
+
+             Run "'"$AMCLI"' -h" to see the list of the options
+ ##########################################################################
+'
+}
+
+# INSTALL "AM" SYSTEM-WIDE
+_install_am() {
+	mkdir -p "$HOME"/.cache
```
Comment (Samueru-sama) [inline AM-INSTALLER:71]: This has to be `${XDG_CACHE_HOME:-$HOME/.cache}` instead of `$HOME/.cache`

**Comment by ivan-hc** [comment]
The "INSTALL" script for "AM"

https://github.com/user-attachments/assets/857d28d4-d2ae-42a0-9fe2-95fd62d48d65

Istantanea_2024-08-04_16-59-11

**Comment by ivan-hc** [comment]
Messages are now less invasive during the installation.

Now I think that all we need is to update the README

**Comment by ivan-hc** [comment]
We are ready

**Pull request merged**